### PR TITLE
Redirect stderr to /dev/null of lsf conf queries

### PR DESCRIPTION
**Pull request by juanesarango**
This PR fixes  https://github.com/DataBiosphere/toil/issues/3750 

I tracked and fixed the issue with the help of @ddomenico 

## Changelog Entry
To be copied to the [draft changelog](https://github.com/DataBiosphere/toil/wiki/Draft-Changelog) by merger:

### LSF
* LSF users don't get errors from `lsadmin` and `bparams` printed in their logs anymore.
## Reviewer Checklist

 * [x] Make sure it is coming from `issues/XXXX-fix-the-thing` in the Toil repo, or from an external repo.
    * [x] If it is coming from an external repo, make sure to pull it in for CI with:
        ```
        contrib/admin/test-pr otheruser theirbranchname issues/XXXX-fix-the-thing
        ```
    * [ ] If there is no associated issue, [create one](https://github.com/DataBiosphere/toil/issues/new).
* [x] Read through the code changes. Make sure that it doesn't have:
    * [x] Addition of trailing whitespace.
    * [x] New variable or member names in `camelCase` that want to be in `snake_case`.
    * [x] New functions without [type hints](https://docs.python.org/3/library/typing.html).
    * [x] New functions or classes without informative docstrings.
    * [x] Changes to semantics not reflected in the relevant docstrings.
    * [x] New or changed command line options for Toil workflows that are not reflected in `docs/running/{cliOptions,cwl,wdl}.rst` 
    * [x] New features without tests.
* [x] Comment on the lines of code where problems exist with a review comment. You can shift-click the line numbers in the diff to select multiple lines.
* [ ] Finish the review with an overall description of your opinion.

## Merger Checklist

* [x] Make sure the PR passes tests.
* [x] Make sure the PR has been reviewed **since its last modification**. If not, review it.
* [ ] Merge with the Github "Squash and merge" feature.
    * [ ] If there are multiple authors' commits, add [Co-authored-by](https://github.blog/2018-01-29-commit-together-with-co-authors/) to give credit to all contributing authors.
* [ ] Copy its recommended changelog entry to the [Draft Changelog](https://github.com/DataBiosphere/toil/wiki/Draft-Changelog).
* [ ] Append the issue number in parentheses to the changelog entry.

### CODE REVIEW

File: src/toil/batchSystems/lsfHelper.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def apply_bparams(fn):
     """
     cmd = ["bparams", "-a"]
     try:
-        output = subprocess.check_output(cmd).decode('utf-8')
+        output = subprocess.check_output(cmd, stderr=subprocess.DEVNULL).decode('utf-8')
```
Comment (mr-c) [inline src/toil/batchSystems/lsfHelper.py:112]: It would be nice if in debug mode we didn't throw this away.

**Review by mr-c — CHANGES_REQUESTED** [review]
Thank you for reporting your issue and providing a fix! We should not throw these messages away while in debug mode. I can make that change if that would be easier.

**Comment by juanesarango** [comment]
Something like this works:
```python
try:
    output = subprocess.check_output(cmd, stderr=subprocess.STDOUT).decode('utf-8')
except subprocess.CalledProcessError as exc:
    logger.debug(exc.output.decode())
    return None
```
@mr-c Is this what you are suggesting?

**Comment by mr-c** [comment]
> Something like this works:
> ```python
> try:
>     output = subprocess.check_output(cmd, stderr=subprocess.STDOUT).decode('utf-8')
> except subprocess.CalledProcessError as exc:
>     logger.debug(exc.output.decode())
>     return None
> ```
> @mr-c Is this what you are suggesting?
> 

Sure! Or use `logger.isEnablefFor(logging.DEBUG)` to toggle setting `stderr` one way or the other

**Review by mr-c — APPROVED** [review]

**Comment by adamnovak** [comment]
OK, I've pulled this into `issues/3750-lsf-queries-to-devnull` for CI testing.

**Pull request merged**